### ISSUE TITLE
`Examples/Geometry/VR` fails to enter VR on first request

### ISSUE THREAD

**Issue by tbirdso** [Kitware/vtk-js#2136]
### High-level description

The "Send to VR" button in `Examples/Geometry/VR` fails to find the connected VR device for rendering the scene.

The problem can be worked around on Firefox by setting the browser to remember allowing access to the VR device and reloading the webpage.

### Steps to reproduce

 * Go to https://kitware.github.io/vtk-js/examples/VR.html
 * Open Web Developer Tools / browser console window
 * Click on "Send to VR" button

### Detailed behavior

 * Browser requests permission to access VR device (Firefox)
 * Console logs exception:
 ```
index.js:280 Uncaught TypeError: Cannot read property 'capabilities' of null
    at Object.publicAPI.startVR (index.js:280)
    at HTMLButtonElement.<anonymous> (index.js:99)
```
 * Inspecting `index.js:280` in console shows that `model.vrDisplay` is `null` when it should have value
 ```
if (model.vrDisplay.capabilities.canPresent) {
```
 * Scene does not launch into VR

Note that [Chrome Developer Tools](https://developer.oculus.com/documentation/web/browser-remote-debugging/) must be used to view the Oculus Browser console. 

### Expected behavior

 * Scene launches in headset and VTK output of cone is displayed in VR on first try

### Environment
- Config 1:
  - **vtk.js**: v20.2.1
  - **OS**: Windows 10
  - **Browser**: Firefox Desktop v93.0 (64-bit)
- Config 2:
  - **vtk.js**: v20.2.1
  - **OS**: Oculus Quest (v30.1)
  - **Browser**: Oculus Browser (v16.6)

**Comment by daker** [comment]
@birdso should this issue be closed?

**Comment by tbirdso** [comment]
No, this has not been fixed as far as I am aware and should not be closed without a corresponding change.

The issue is low impact and may be resolved with upcoming XR development. No harm in keeping it open.
